src/test/stateful_test_utils.hpp

```diff
@@ -26,12 +26,22 @@
 
 #include "../sequence.hpp"
 #include "../sequence_manager.hpp"
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wall"
+#include "tensorflow_serving/apis/prediction_service.grpc.pb.h"
+#pragma GCC diagnostic pop
 #include "mock_iinferrequest.hpp"
 
 #include <gmock/gmock-generated-function-mockers.h>
 
 using namespace InferenceEngine;
 
+const uint32_t NO_CONTROL_INPUT = 0;
+const uint32_t SEQUENCE_START = 1;
+const uint32_t SEQUENCE_END = 2;
```

Comment:
I think we define those values somewhere in our includes above (sequence.hpp or sequence_manager.hpp). 
They are in ovms namespace so in this file you would have to access it via ovms::NO_CONTROL_INPUT etc. 
No need for those variables here.